extensions/Khronos/KHR_materials_common_v2/README.md

```diff
@@ -0,0 +1,250 @@
+
+For all technique computations, the following definitions are assumed:
+*(all vectors normalized)*
+
+* `diffuseTerm` – diffuseFactor * diffuseTexture
+    * *Reflected diffuse color of the material.*
+* `specularTerm` – specularFactor * specularShininessTexture.rgb
+    * *Specular color of the material.*
+* `shineTerm` – shininessFactor * specularShininessTexture.a
+    * *Shininess component of the specular color.*
+* `emissiveTerm` – emissiveFactor * emissiveTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `occlusionTerm` – occlusionFactor * occlusionTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `aL` – Ambient light
+    * *Constant sum of all ambient light contributions from the scene.*
+* `N` – Normal vector
+* `L` – Light vector
+* `I` – Eye vector
+* `H` – Half-angle vector
+    * *Calculated as halfway between the unit Eye and Light vectors, using the equation `H = normalize(I+L)`.*
+
+### Constant
+
+The `CONSTANT` technique defines shading that is independent of lighting, or
+unlit. No extension properties are included with Constant materials, but
+emissive color (from the core glTF 2.0 specification) is used.
+
+The displayed color is calculated as:
+
+```
+color = emissiveTerm + occlusionTerm
+```
+
+> **Implementation Note**: The constant shading model is visually identical to
+> Lambert or Blinn-Phong models in which all unused factors are set to zero.
+> Accordingly, engines may implement only the Blinn-Phong model and use it to
+> emulate simpler shading models if preferred.
+
+<details>
+    <summary>Example: Constant</summary>
+The following example defines a Constant material with an emissive red color.
+
+```json
+    "materials": [
+        {
+            "name": "red_constant_material",
+            "emissiveFactor": [ 1.0, 0.0, 0.0 ],
+            "extensions": {
+                "KHR_materials_cmnBlinnPhong": {
+                    "technique": "CONSTANT"
+                }
+            }
+        }
+    ]
+```
+</details>
+
+### Lambert
+
+The `LAMBERT` technique defines a material shaded using Lambert shading. The
+result is based on Lambert’s Law, which states that when light hits a rough
+surface, the light is reflected in all directions equally. The reflected
+color is calculated as:
+
+```
+color = emissiveTerm + occlusionTerm * diffuseTerm * max(N * L, 0)
+```
+
+> **Implementation Note**: The Lambert shading model is visually identical to a
+> Blinn-Phong model in which all unused factors are set to zero. Accordingly,
+> engines may implement only the Blinn-Phong model and use it to emulate
+> simpler shading models if preferred.
+
+<details>
+    <summary>Example: Lambert</summary>
+The following example defines a Lambert shaded material with a diffuse texture,
+moderate shininess and red specular highlights. 
+
```

Comment:
Looks like a copy-and-paste error. Maybe just "The following example defines a Lambert shaded material with a diffuse texture." or some-such?